components/mixins/experience-event/experienceevent-alert-impressions.schema.json

```diff
@@ -0,0 +1,48 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/mixins/experienceevent-alert-impressions",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Alert Impressions",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"],
+  "description": "Alert information such as ad impressions and ad clicks. ",
+  "definitions": {
+    "alert-impressions": {
+      "properties": {
+        "xdm:alertImpressions": {
+          "xdm:alertClicks": {
```

Comment:
Is this and other properties events/actions ? Should they be "measures" ?

Reply:
It is in others.  I've updated it.